packages/nodes-base/nodes/MongoDb/MongoDbProperties.ts

```diff
@@ -151,6 +151,17 @@ export const nodeProperties: INodeProperties[] = [
 				placeholder: '{ "field": -1 }',
 				description: 'A JSON that defines the sort order of the result set',
 			},
+			{
+				displayName: 'Projection (JSON Format)',
+				name: 'projection',
+				type: 'json',
+				typeOptions: {
+					rows: 2,
+				},
+				default: '{}',
+				placeholder: '{ "field": -1 }',
```

Comment:
Does this placeholder work for this field?

Reply:
Good point, let me improve 👍